RNGNewAuraNotifier/Core/Notification/DiscordNotificationService.cs

```diff
@@ -26,7 +26,7 @@ public static async Task Notify(string title, string message, VRChatUser? vrchat
             Description = message,
             Footer = new EmbedFooterBuilder
             {
-                Text = $"{AppConstant.AppName}{AppConstant.AppVersion.Major}.{AppConstant.AppVersion.Minor}.{AppConstant.AppVersion.Build}",
+                Text = $"{AppConstants.AppName}{AppConstants.AppVersionString}",
```

Comment:
申し訳ない、これ元々から抜けちゃってたんですけど、アプリケーション名とバージョンテキストの間のスペースが抜けてます。

```suggestion
                Text = $"{AppConstants.AppName} {AppConstants.AppVersionString}",
```